src/style-installer.js

```diff
@@ -249,29 +244,21 @@ export function makeBodyVisible(doc, opt_waitForServices) {
   };
   try {
     waitForBody(doc, () => {
```

Comment:
Using `waitForBodyPromise` would allow us drop the try-catch.

Reply:
+1

Reply:
Done.